migrations/versions/rev-2020-12-28-11:07:56-7374b18af581_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: 7374b18af581
+Revises: b0c55c767022
+Create Date: 2020-12-28 11:07:56.703331
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '7374b18af581'
+down_revision = 'b0c55c767022'
+
+
+def upgrade():
+    op.execute("UPDATE custom_forms SET is_public=True WHERE form='session';")
```

Comment:
All session fields are public?

Reply:
yeah I think it was said in meeting that session details should be public.

Reply:
Make only those session fields public which are shown in UI currently